microsoft-dotnet/3.1/modules/ROOT/pages/microsoft-dotnet-connector-examples.adoc

```diff
@@ -0,0 +1,119 @@
+= Microsoft .NET Connector 3.1 Examples - Mule 4
+
+The following use case...
+*Reviewers - please summarize what this use case does and also look at the example carefully. I added more detailed steps.*
+
+image::microsoft-dotnet-use-case.png[Studio Flow Diagram]
+
+. Create a new Mule project in Studio.
+. In the *Mule Palette* view, search for `http` and select the *Listener* operation.
+. Drag the *Listener* operation onto the Studio canvas.
+. In the *Listener* configuration, click *+* next to the *Connector configuration* field to add a global element.
+. Set the *Host* field to `localhost`.
+. Set the *Path* field to `sumOfDigits`.
```

Comment:
The `Path` field is outside the `Connector configuration` dialog which also contains a `Base path` field. We should add something here like: `Close configuration dialog and then set the path field located in General section of the listener`